src/index.ts

```diff
@@ -119,14 +125,12 @@ export default class ServerlessConventions {
 
     // If there were errors detected, print out a list and throw an error
     if (errors.length !== 0) {
-      errors.forEach((error) => {
-        this.serverless.cli.log(chalk.red(error));
-      });
-
-      throw Error('Serverless conventions validation failed');
+      throw new this.serverless.classes.Error(
+        'Serverless conventions validation failed!\n' + errors.join('\n')
```

Comment:
We can't use chalk to colour these? :disappointed: Unfortunately I feel like the errors will be easily ignored if not red... (based on past experience). Is there any harm in logging in twice: one in colour and the other using this?